4-esqueleto/README 12.11.39.md

```diff
@@ -0,0 +1,140 @@
+
+![tuspropuestas](/4-esqueleto/tuspropuestas.png)
+
+El tic verde indica que el administrador del sitio web ha revisado los archivos y que le ha dado el visto bueno para poder publicarlo. La interrogación significa que el archivo todavía está en la fase de revisión. Y finalmente, la cruz roja significa que el archivo no se va a publicar. 
+
+![tuspublicaciones](/4-esqueleto/tuspublicaciones.png)
+
+En el apartado “Tus publicaciones” se encuentran aquellos archivos que han sido revisados y aceptados por el administrador. En la parte derecha de cada archivo aparece la opción “Acceder” que permite acceder a la publicación en cuestión ya publicada a disposición de cualquier usuario. 
+
+En este apartado, nuestra página contará con un elemento de localización, que permitirá a cada usuario saber dónde se encuentra en cada momento durante su navegación, pudiendo volver hacia atrás a través de esta ruta con hipervínculos.
+
+Además, la página también contará con un Mapa del Sitio. Con él, permitiremos al usuario en cuestión saltar a la página que desee.
+
+
+## DISEÑO DE LA NAVEGACIÓN 
+
+### Organización de ruta
+En este apartado, nuestra página contará con un elemento de localización, que permitirá a cada usuario saber dónde se encuentra en cada momento durante su navegación, pudiendo volver hacia atrás a través de esta ruta con hipervínculos.
+
+Además, la página también contará con un Mapa del Sitio. Con él, permitiremos al usuario en cuestión saltar a la página que desee.
+
+### Navegación 
+Así mismo, también hemos decidido que nuestra página sea de navegación a través de barras y pestañas. Concretamente, hemos decidido que lleve un menú dinámico dentro de ella. 
+
+Dentro de cada temática existirán subtemas que podrán ser seleccionados a través de pop up menus. Esto hará que nuestra web posea un aspecto moderno y sencillo de llevar.
+
+
+## DISEÑO DE LA INFORMACIÓN 
+
+#### Variedades de Etiquetas
+En la Web, nos encontramos regularmente con etiquetas en un formato textual. Trataremos de usar las etiquetas de texto ya que siguen siendo las más comunes, a pesar de que los contenidos de carácter visual hayan ganado mucho peso en la web:
+
+#### Enlaces contextuales
+Una buena forma de entablar relaciones textuales entre los contenidos publicados en el sitio web reside sin duda en establecer hipervínculos en determinados contenidos, con el objetivo de conectar varias páginas relacionadas entre sí a través de este proceso de etiquetado. Esta fórmula permite conectar varios contenidos similares, ayudando a comprender un determinado contenido a través de la contextualización que ofrece la unión de varias páginas en una sola. Puede resultar un proceso muy sencillo de efectuar y muy eficaz en lo que se refiere al proceso de ejecución técnica, si bien es cierto que puede resultar un inconveniente para los redactores de los contenidos, que deberían tener en cuenta este factor a la hora de elaborar sus respectivos contenidos.  
```

Comment:
¿Y cómo lo vais a implementar en vuestro sitio? Esta aportación teórica no tiene mucho sentido si no explicáis cómo se aplica en vuestro caso concreto.